Makefile

```diff
@@ -22,8 +25,7 @@ checkpy: all
 checkvim: all
 	vint autoload
 
-test:
-	test/run.sh
+vim/test:
```

Comment:
This looks wrong.. should run `test/run.sh` still probably?

Reply:
oops... fixed.

Reply:
ah also `fast-test` must reference this task...